superset/connectors/sqla/models.py

```diff
@@ -900,20 +924,40 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
         order_desc: bool = True,
     ) -> SqlaQuery:
         """Querying any sqla table from this common interface"""
-        template_kwargs = {
-            "from_dttm": from_dttm.isoformat() if from_dttm else None,
-            "groupby": groupby,
-            "metrics": metrics,
-            "row_limit": row_limit,
-            "row_offset": row_offset,
-            "to_dttm": to_dttm.isoformat() if to_dttm else None,
-            "filter": filter,
-            "columns": [col.column_name for col in self.columns],
-        }
-        template_kwargs.update(self.template_params_dict)
+
         extra_cache_keys: List[Any] = []
-        template_kwargs["extra_cache_keys"] = extra_cache_keys
+        is_sip_38 = is_feature_enabled("SIP_38_VIZ_REARCHITECTURE")
```

Comment:
This feature flag has been removed a while ago, so you probably need to double check that all changes here are still relevant.